### PR TITLE
Add more translations of library/io

### PR DESCRIPTION
**Pull request by cschan1828**
Add more translations of `library/io`

Note: This commit was co-authored with AI.

### CODE REVIEW

File: library/io.po [hunk 1]
```diff
@@ -652,63 +652,76 @@ msgstr "為了方便起見，允許多次呼叫這個方法；然而，只有第
...
-msgstr ""
+msgstr "如果串流已關閉，則為 ``True``。"
 
 #: ../../library/io.rst:366
 msgid ""
 "Return the underlying file descriptor (an integer) of the stream if it "
 "exists.  An :exc:`OSError` is raised if the IO object does not use a file "
 "descriptor."
 msgstr ""
+"如果串流存在，則回傳其底層的檔案描述符（一個整數）。如果 IO 物件不使用檔案描"
```
Comment (mattwang44) [inline library/io.po]: descriptor 在 glossary 頁面被翻成「描述器」
https://docs.python.org/zh-tw/3/glossary.html#term-descriptor

File: library/io.po [hunk 2]
```diff
@@ -652,63 +652,76 @@ msgstr "為了方便起見，允許多次呼叫這個方法；然而，只有第
...
 msgstr ""
+"如果適用，清空串流的寫入緩衝區。對於唯讀和非阻塞串流，此操作不會執行任何操"
+"作。"
 
 #: ../../library/io.rst:377
 msgid ""
 "Return ``True`` if the stream is interactive (i.e., connected to a terminal/"
 "tty device)."
-msgstr ""
+msgstr "如果串流是互動式的（即，連接到終端機/tty 設備），則回傳 ``True``。"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
msgstr "如果串流是互動式的（即連接到終端機/tty 設備），則回傳 ``True``。"
```

File: library/io.po [hunk 3]
```diff
@@ -652,63 +652,76 @@ msgstr "為了方便起見，允許多次呼叫這個方法；然而，只有第
...
 "*hint* values of ``0`` or less, as well as ``None``, are treated as no hint."
-msgstr ""
+msgstr "*hint* 值為 ``0`` 或更小，以及 ``None``，都被視為沒有提供 hint。"
 
 #: ../../library/io.rst:403
 msgid ""
 "Note that it's already possible to iterate on file objects using ``for line "
 "in file: ...`` without calling :meth:`!file.readlines`."
 msgstr ""
+"請注意，已經可以使用 ``for line in file: ...`` 在檔案物件上進行迭代，而不一定"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"請注意，已經可以使用 ``for line in file: ...`` 在檔案物件上進行疊代，而不一定"
```

File: library/io.po [hunk 4]
```diff
@@ -652,63 +652,76 @@ msgstr "為了方便起見，允許多次呼叫這個方法；然而，只有第
...
+msgstr "從串流讀取並回傳一行。如果指定了 *size*，則最多讀取 *size* 個位元組。"
 
 #: ../../library/io.rst:390
 msgid ""
 "The line terminator is always ``b'\\n'`` for binary files; for text files, "
 "the *newline* argument to :func:`open` can be used to select the line "
 "terminator(s) recognized."
 msgstr ""
+"對於二進位檔案，行結束符總是 ``b'\\n'``；對於文字檔案，可以使用 :func:`open` "
+"函式的 *newline* 參數來選擇識別的行結束符號。"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"函式的 *newline* 引數來選擇識別的行結束符號。"
```

**Comment by cschan1828** [comment]
Thank you for your review. I collected your feedback to my AI model.

File: library/io.po [hunk 5]
```diff
@@ -652,63 +652,76 @@ msgstr "為了方便起見，允許多次呼叫這個方法；然而，只有第
...
+msgstr "如果串流已關閉，則為 ``True``。"
 
 #: ../../library/io.rst:366
 msgid ""
 "Return the underlying file descriptor (an integer) of the stream if it "
 "exists.  An :exc:`OSError` is raised if the IO object does not use a file "
 "descriptor."
 msgstr ""
+"如果串流存在，則回傳其底層的檔案描述器（一個整數）。如果 IO 物件不使用檔案描"
+"述符，則會引發一個 :exc:`OSError` 例外。"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"述器，則會引發一個 :exc:`OSError` 例外。"
```

**Review by mattwang44 — APPROVED** [review]

**Pull request merged**